packages/rich-text-from-markdown/src/index.ts

```diff
@@ -117,6 +132,33 @@ const buildGenericBlockOrInline = async (
   ];
 };
 
+const buildTableCell = async (
+  node: MarkdownNode,
+  fallback: FallbackResolver,
+  appliedMarksTypes: string[],
+): Promise<Array<Block>> => {
+  const content = await mdToRichTextNodes(node.children, fallback, appliedMarksTypes);
+
+  /**
+   * We should only support texts inside table cells.
+   * Some markdowns might contain html inside tables such as <ul>, <blockquote>, etc
+   * but they are pretty much filtered out by markdownNodeTypes and nodeContainerTypes variables.
```

Comment:
But `markdownNodeTypes` also includes ul, blockquote, etc. that aren't supported inside a table... seems to me like we'd have to maintain a list of node types supported inside tables and only insert plaintext or raw markdown (not sure) for any other type.

Reply:
The markdown parser doesn't return block elements inside list. If I do:

```
| Header 1 | Header 2 |
|----------|----------|
| > Test   | * List   |
```

It will return `> Test` and `* List` as a `text`. 

I've also tried:

```
| Header 1 | Header 2 |
|----------|----------|
| <blockquote>Test</blockquote>   | * List   |
```

It returns this as an `html` block then the text inside as a `text`, we ignore the html block.

Reply:
Also, `content` comes from `node.children`, which is will have the children from the table cell. Which is pretty much `text`.